front/src/reducers/osrdconf/operationalStudiesConf/powerRestrictionReducer.ts

```diff
@@ -10,234 +10,207 @@ import { addElementAtIndex } from 'utils/array';
 import type { OperationalStudiesConfState } from '.';
 import { addPathStep, cleanPathSteps, isRangeCovered, updateRestrictions } from './utils';
 
-export type PowerRestrictionReducer<S extends OperationalStudiesConfState> = {
-  ['updatePowerRestrictionRanges']: CaseReducer<S, PayloadAction<PowerRestriction[]>>;
-  ['upsertPowerRestrictionRanges']: CaseReducer<
-    S,
-    PayloadAction<{ from: PathStep; to: PathStep; code: string }>
-  >;
-  ['cutPowerRestrictionRanges']: CaseReducer<S, PayloadAction<{ cutAt: PathStep }>>;
-  ['deletePowerRestrictionRanges']: CaseReducer<S, PayloadAction<{ from: PathStep; to: PathStep }>>;
-  ['resizeSegmentEndInput']: CaseReducer<
-    S,
-    PayloadAction<{
-      firstRestriction: PowerRestriction;
-      secondRestriction?: PowerRestriction;
-      newEndPathStep: PathStep;
-    }>
-  >;
-  ['resizeSegmentBeginInput']: CaseReducer<
-    S,
-    PayloadAction<{
+const powerRestrictionReducer = {
```

Comment:
is there any significant changes here or is it just moving code around?

Reply:
It's just moving code around, no behavior change :+1: